app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php

```diff
@@ -0,0 +1,168 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Directory\Setup\Patch\Data;
+
+use Magento\Directory\Setup\DataInstaller;
+use Magento\Directory\Setup\DataInstallerFactory;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+
+/**
+ * Add Russian Regions
+ */
+class AddDataForRussia implements DataPatchInterface
+{
+    /**
+     * @var ModuleDataSetupInterface
+     */
+    private $moduleDataSetup;
+
+    /**
+     * @var DataInstallerFactory
+     */
+    private $dataInstallerFactory;
+
+    /**
+     * @param ModuleDataSetupInterface $moduleDataSetup
+     * @param DataInstallerFactory $dataInstallerFactory
+     */
+    public function __construct(
+        ModuleDataSetupInterface $moduleDataSetup,
+        DataInstallerFactory $dataInstallerFactory
+    ) {
+        $this->moduleDataSetup = $moduleDataSetup;
+        $this->dataInstallerFactory = $dataInstallerFactory;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function apply()
+    {
+        /** @var DataInstaller $dataInstaller */
+        $dataInstaller = $this->dataInstallerFactory->create();
+        $dataInstaller->addCountryRegions(
+            $this->moduleDataSetup->getConnection(),
+            $this->getDataForRussia()
+        );
+    }
```

Comment:
```suggestion
        return $this;
    }
```

Reply:
AddDataForAustralia - no return $this;
AddDataForBelgium  - no return $this;
AddDataForChina  - no return $this;
AddDataForColombia  - no return $this;
.....

Reply:
@luckyraul We are not in the kindergarten to refer to other bad examples. You have the failing test so please make it green